src/main/deploy/constants/ScoringSetpoints.json

```diff
@@ -123,12 +123,12 @@
   "netWarmup": {
     "name": "netWarmup",
     "elevatorHeight": {
-      "value": 0.25,
+      "value": 1.55,
```

Comment:
Just a note: I think this might be too low - the wrist snap needs to happen at the top of the elevator. Obviously test and tune in shop as needed